src/modules/kvs/kvstxn.c

```diff
@@ -55,7 +55,8 @@ struct kvstxn {
     json_t *ops;
     json_t *keys;
     json_t *names;
-    int flags;
+    int flags;                  /* kvs flags from request caller */
+    int api_flags;       /* special kvstxn api flags */
```

Comment:
I'm not sure the `api_` prefix is communicating that it's an internal flag.  Would `internal_flags` be better?

Reply:
i wasn't the biggest fan of "api_flags" either.  The issue was there is already an "internal_flags" used internally within the `kvstxn` api (internal means not externally exposed to users).  

perhaps could be called `kvstxn_flags`?  perhaps the other flags needs to renamed to `op_flags`?  lemme think about it.

Reply:
ahh i had a good idea, the "internal_flags" didn't really have to be bitmasks, they were just there as "bools" for certain conditions.  So I replaced them with a bunch of bool flags.  Then api_flags could be "internal_flags" instead.